### PR TITLE
Mobile app inventory record view

### PR DESCRIPTION
**Pull request by shackyshell**

### CODE REVIEW

File: mobile/services/StocktakingService.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        console.error(error);
+    }
+}
+
+export async function updateStocktaking(id, is_checked) {
+    let headers = null
+    await setHeaders().then((res) => headers = res);
+    try {
+        let response = await fetch(
+            `${API_URL}/api/inventories/supplies/${id}`, {
```
Comment (r-kaminski) [inline mobile/services/StocktakingService.js]: This endpoint has been changed, now it's "api/inventories/**\<id of inventory report\>**/supplies/\<id of supply\>"

**Review by r-kaminski — CHANGES_REQUESTED** [review]
Endpoint used in updateStocktaking was changed

File: mobile/services/StocktakingService.js (hunk 1 above)
Comment (shackyshell) [reply mobile/services/StocktakingService.js]: poprawione

**Review by r-kaminski — CHANGES_REQUESTED** [review]
1. Stocktaking create stage has [this](https://imgur.com/a/tOo8h0C) weird behavior, that takes whatever the user inputs and displays it at the left-top corner of the screen, therefore moving input form slightly down. Is this intended behavior?
2. Inventory record/report details view should display name of the supply, description and it's checkbox. So basically re-usage of regular supplies list view [like this](https://imgur.com/a/5BicX4s). For this to work properly, the return values of report details should be fixed first: https://github.com/nokia-wroclaw/innovativeproject-inventory-of-supplies/issues/98. 
By fixing this backend issue, while accessing report details we'll get list of inventory supplies with only some of the details (name, description, is_checked status, ID and it's report ID). Name, description and is_checked will be displayed for each supply, and also we'll be able to pass it's ID and report ID to **GET /api/inventories/<inventory_id>/supplies/<supply_id>** endpoint and see the rest of the details, just like regular supplies details view.
3. Same view discussed in point 2. shows only 10 of the stocktaking's supplies and there's no way to see the rest of them. I suppose that pagination will be introduced in https://github.com/nokia-wroclaw/innovativeproject-inventory-of-supplies/issues/64 and is not subject of this PR, right?

**Review by MarekPokropinski — APPROVED** [review]

**Comment by shackyshell** [comment]
Mobile app inventory record view #81 https://github.com/nokia-wroclaw/innovativeproject-inventory-of-supplies/issues/81

**Pull request merged**